src/FixedDataTableCell.js

```diff
@@ -245,14 +245,18 @@ var FixedDataTableCell = createReactClass({
       var columnResizerStyle = {
         height
       };
+      function suppress(event) {
+        event.preventDefault();
+        event.stopPropagation();
+      };
       columnResizerComponent = (
         <div
           className={cx('fixedDataTableCellLayout/columnResizerContainer')}
           style={columnResizerStyle}
           onMouseDown={this._onColumnResizerMouseDown}
           onTouchStart={this.props.touchEnabled ? this._onColumnResizerMouseDown : null}
-          onTouchEnd={this.props.touchEnabled ? e => e.stopPropagation() : null}
```

Comment:
@SirGandal can you give this a quick look that it doesn't break your use case?

Reply:
@wcjordan sorry for the late reply. That should be fine.
@davidhaimson out of curiosity, what's the reason for preventDefault in onTouchEnd and onTouchMove?

Reply:
First thing I tried was adding `event.preventDefault()` in the `_onColumnResizerMouseDown` method. That made a big difference on the Android devices where I was testing this. Without it, trying to resize the columns scrolled the table body left/right. I made the corresponding change here for consistency.

Because of my injury, I am working from home, and I don't have the physical devices to test on. I tried removing all the `preventDefault` calls, and it still works, on all the platforms, except I had to test Android and iOS in emulators. Now I don't know what to think.

I was able to test the version with all `preventDefault` calls in the modified demo app on a physical iPhone and Android phone, and they worked fine. My intuition is that these calls are at the very least harmless, and the one in `_onColumnResizerMouseDown` is necessary. But I can't prove it until I get back to work on site. Not sure when that will be - a couple of weeks, probably.

Reply:
What do you want to do here?  Should we wait til you're back on site and can test fully before finalizing this?

Reply:
We can do that, but as I say, it'll be a couple of weeks. I already tested it fully (with the harmless, but possibly superfluous `preventDefault` calls). My recommendation would be to merge it in its present form. When I am back on-site, we can do a PR trimming it down to the minimum necessary.

Reply:
Sounds good.  I'll merge and look to release over the weekend.  Thanks

Reply:
You're a prince! Thanks.